.circleci/config.yml

```diff
@@ -117,6 +117,21 @@ jobs:
       - store_test_results:
           path: ui/test-results
 
+  test-ember:
+    docker:
+      - image: *NODE_IMAGE
+    steps:
+      - checkout
+      - restore_cache:
+          key: *YARN_LOCK_CACHE_KEY
+      - attach_workspace:
+          at: .
+      - run:
+          name: Test Ember
+          command: |
+            set -eux -o pipefail
+            make ember-ci-test
```

Comment:
Might want to rename this Makefile target to `test-browserstack`? Would require the Travis config to be updated too. (CC: @meirish for thoughts)

Reply:
Ah yeah that's a good call - +1 on that.

Reply:
Done, I've called it `test-ui-browserstack` I hope that's ok, matches the desired name for the check in github, so it's the same name for the same thing everywhere now. Travis also updated and is passing these tests for the right reason (because the tests passed). CircleCI still needs config, but I'm blocked on having access to the browserstack creds, so putting this on hold for now. I've contacted helpdesk @jfreda @meirish